lib/rucio/client/opendataclient.py

```diff
@@ -172,7 +173,8 @@ def update_opendata_did(
             name: The name of the DID.
             state: The new state to set for the DID.
             meta: Metadata to update for the DID. Must be a valid JSON object.
-            doi: DOI to associate with the DID. Must be a valid DOI string (e.g., "10.1234/foo.bar").
+            doi: DOI to associate with the DID. Must be a valid DOI string (e.g., "10.1234/foo.bar") and unique across all DIDs.
+            record_id: The record ID of the DID to update. This can be used to cross-reference with external systems. Must be unique across all DIDs.
```

Comment:
Do you have a reference for this Open Data record ID concept? What's the difference between this and a DOI (I'm assuming all OpenData DID should have a unique DOI anyway?)